packages/lesswrong/components/review/BestOfLessWrongAdmin.tsx

```diff
@@ -0,0 +1,182 @@
+
+  return post && <div key={post._id} className={classes.root}>
+    {imageThumbnail && <img src={imageThumbnail} />}
+    <div className={classes.imageInfo}>
+      <h2>
+        <Link target="_blank" to={postGetPageUrl(post)}>{post.title}</Link>
+      </h2>
+      <GenerateImagesButton 
+        postId={post._id}
+        allowCustomPrompt={true}
+        buttonText="Generate More Images"
+        onComplete={refetchImages}
+      />
+      <PostWithArtGrid key={post._id} post={post} images={images} defaultExpanded={false} />
+    </div>  
+  </div>
+}
+
+const styles = defineStyles("BestOfLessWrongAdmin", (theme: ThemeType) => ({
+  root: {
+    paddingLeft: 50,
+    paddingRight: 50,
+    ...theme.typography.body2,
+    '& h2': {
+      marginTop: 10,
+      marginBottom: 0,
+      position: 'sticky',
+      top: 0,
+      zIndex: theme.zIndexes.bestOfLessWrongAdminHeader,
+      backgroundColor: theme.palette.background.default,
+    }
+  },
+  post: {
+    ...theme.typography.body2,
+    ...theme.typography.commentStyle,
+    display: 'flex', 
+    gap: '10px',
+    alignItems: 'flex-start',
+    marginBottom: 30,
+    '& h3': {
+      marginBottom: 6,
+    }
+  },
+  postWrapper: {
+    display: "flex",
+    gap: '10px',
+    flexWrap: "wrap",
+    textWrap: "wrap",
+  },
+  generateButton: {
+    padding: '10px 0',
+    display: 'block',
+    color: theme.palette.primary.dark,
+    backgroundColor: 'transparent',
+    border: 'none',
+    fontWeight: 'bold',
+    cursor: 'pointer',
+    '&:disabled': {
+      opacity: 0.5,
+      cursor: 'not-allowed',
+    }
+  },
+  loadingText: {
+    marginLeft: 10,
+    color: theme.palette.grey[600],
+    fontStyle: 'italic',
+  }
+}));
+
+/* 
+Display the art for all ReviewWinners created in a given year,
+making it easier to quickly make a call for each post.
+
+(Note, because it was a bit annoying to do the fetch otherwise, this is using *createdAt*
+date rather than ReviewYear, i.e. 2 years after a ReviewYear)
+
+*/
+export const BestOfLessWrongAdmin = () => { 
+  const classes = useStyles(styles);
+  const { Loading } = Components;
+  
+  const currentUser = useCurrentUser();
+
+  const { data, loading: reviewWinnersLoading } = useQuery(gql`
+    query GetAllReviewWinners {
+      GetAllReviewWinners {
+        ...PostsTopItemInfo
+      }
+    }
+    ${fragmentTextForQuery('PostsTopItemInfo')}
+  `);
+  const reviewWinners = data?.GetAllReviewWinners ?? [];
+  const reviewWinnersWithoutArt = reviewWinners.filter((reviewWinner: PostsTopItemInfo) => !reviewWinner.reviewWinner?.reviewWinnerArt);
+
+  const { params: { year } } = useLocation()
+
+  const { results: images, loading: imagesLoading, refetch: refetchImages } = useMulti({
+    collectionName: 'ReviewWinnerArts',
+    fragmentName: 'ReviewWinnerArtImagesForYear',
+    terms: {
+      view: 'allForYear',
+      year: parseInt(year),
+      limit: 5000,
+    },
+    skip: !year,
```

Comment:
If a page is admin-gated on the client (as it is below) then I'd feel better about having the user admin check on the skip, too.  (Mostly for bots rather than real users stumbling on the page.)